static/opensearch.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
+                      xmlns:moz="http://www.mozilla.org/2006/browser/search/">
+    <ShortName>CC Search</ShortName>
+    <Description>Creative Commons search tool</Description>
+    <InputEncoding>UTF-8</InputEncoding>
+    <Image height="16" width="16" type="image/x-icon">
+        "https://ccsearch.creativecommons.org/static/img/cc-icon_large.443ce99.png"
```

Comment:
This image URL doesn't return the image. Shouldn't it be `https://ccsearch.creativecommons.org/static/img/cc-icon_large.png`?

Reply:
you're right, got confused by the name of this file in the static folder